static/js/components/tiles/chart_footer.tsx

```diff
@@ -25,20 +25,21 @@ import { NL_SOURCE_REPLACEMENTS } from "../../constants/app/nl_interface_constan
 import { urlToDomain } from "../../shared/util";
 import { isNlInterface } from "../../utils/nl_interface_utils";
 
-interface ChartFooterPropType {
-  // set of full urls of sources of the data in the chart
-  sources: Set<string>;
-  handleEmbed?: () => void;
+export interface ClickFnType {
```

Comment:
nit: since this is an exported interface, can we make the name more informative. Maybe something about ChartFooter like ChartFooterClickFns or something of that sort.

Also add a comment describing this type?